app/assets/javascripts/discourse/app/components/post/menu.gjs

```diff
@@ -498,6 +499,10 @@ export default class PostMenu extends Component {
   <template>
     {{! The section tag can't be include while we're still using the widget shim }}
     {{! <section class="post-menu-area clearfix"> }}
+    <PluginOutlet
+      @name="post-menu__before"
+      @outletArgs={{hash context=this.context post=@post}}
+    />
```

Comment:
Should we make this a wrapper plugin outlet? Then we get the `__before` and `__after` for free?

Reply:
Yep. I replaced them